### ISSUE TITLE
IDE freezes

### ISSUE THREAD

**Issue by kad-malota** [SERG-Delft/testknight#16]
When opening the "Plugins" settings, with testknight in the searchbox, the whole IDE freezes and can only be terminated using the Windows Task Manager.

**Comment by Teisted** [comment]
I've been trying to recreate the issue but I'm not sure if I'm doing it right. 

![image](https://user-images.githubusercontent.com/26065332/152526173-7cd26cc1-0be8-497f-8668-4a7b4b4d36cb.png)

Does it happen when you have TestKnight in the search bar like in the picture?

**Comment by kad-malota** [comment]
Yes, exactly!

Met vriendelijke groet,

Eric Jan Malotaux
Software engineer

**Comment by Teisted** [comment]
Unfortunately, we still haven't been able to reproduce this issue. 

If the issue is persisting, can you share more information about it? Does this happen when you search for TestKnight anytime you open the settings panel during your IntelliJ session or right after you start your session?

**Comment by kad-malota** [comment]
If I have many workspaces open at the time I try to install TestKnight, IntelliJ freezes the moment I select TestKnight from the Marketplace.
When I close all but one of the workspaces I can install TestKnight. However, when I select TestKnight again in the Plugins-panel, it freezes again and the only way to deinstall it is by deleting the files from the IntelliJ plugins folder.
In the image below, the busy thingy does not rotate; is is frozen too.

***@***.***

Met vriendelijke groet,

Eric Jan Malotaux
Software engineer

**Comment by kad-malota** [comment]
The freeze might be associated with the VPN I am connected to most of the day. Without it, there are no freezes. Would that give you any clue?

Met vriendelijke groet,

Eric Jan Malotaux
Software engineer

**Comment by kad-malota** [comment]
Yes! That's definitly it. IntelliJ unfreezes the moment I disconnect from the VPN.

**Comment by Teisted** [comment]
That's an interesting bug. Does this happen only with TestKnight?
